src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParseTreeTest.java

```diff
@@ -36,7 +36,9 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-public class JavadocParseTreeTest {
+import com.puppycrawl.tools.checkstyle.BaseCheckTestSupport;
+
+public class JavadocParseTreeTest extends BaseCheckTestSupport {
```

Comment:
hmm. Would make the commits more readable yes. The perhaps I will clear out #3796 and #4381 first.